app/assets/stylesheets/components/_my_crimes.scss

```diff
@@ -1,9 +1,8 @@
-// #my-crimes {
-//   color: red;
-// }
-
-.text-primary {
-  text-align: center;
+.container {
```

Comment:
I think we should use a different class name here, and apply it where needed, instead of overwriting the existing container class